packages/Liquid-Core.package/LiquidStartMenu.class/instance/enterParticipantMode.st

```diff
@@ -2,7 +2,7 @@ modes
 enterParticipantMode
 |pollId menu|
 
-pollId := FillInTheBlank request: 'What''s your Poll ID?' initialAnswer: 'mockup'.
+pollId := FillInTheBlank request: 'What''s your Poll ID?' initialAnswer: Clipboard clipboardText asString.
```

Comment:
When I saw this, my initial question was "Can `Clipboard clipboardText` return `nil`?" Had a look at the code, apparently it can't. So this is fine :)

Reply:
Very good thought! But even if it _could_ return `nil`, `nil asString` would just return 'nil'.